.deps/resolved/linux-aarch64_3.12.txt

```diff
@@ -58,7 +57,7 @@ opentelemetry-api @ https://agent-int-packages.datadoghq.com/external/openteleme
 orjson @ https://agent-int-packages.datadoghq.com/external/orjson/orjson-3.11.3-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl#sha256=8b13974dc8ac6ba22feaa867fc19135a3e01a134b4f7c9c28162fed4d615008a
 os-service-types @ https://agent-int-packages.datadoghq.com/external/os-service-types/os_service_types-1.8.0-py3-none-any.whl#sha256=bc0418bf826de1639c7f54b2c752827ea9aa91cbde560d0b0bf6339d97270b3b
 packaging @ https://agent-int-packages.datadoghq.com/external/packaging/packaging-25.0-py3-none-any.whl#sha256=29572ef2b1f17581046b3a2227d5c611fb25ec70ca1ba8554b24b0e69331a484
-paramiko @ https://agent-int-packages.datadoghq.com/external/paramiko/paramiko-4.0.0-py3-none-any.whl#sha256=0e20e00ac666503bf0b4eda3b6d833465a2b7aff2e2b3d79a8bba5ef144ee3b9
+paramiko @ https://agent-int-packages.datadoghq.com/external/paramiko/paramiko-3.5.1-py3-none-any.whl#sha256=43b9a0501fc2b5e70680388d9346cf252cfb7d00b0667c39e80eb43a408b8f61
```

Comment:
is the noticeable _downgrade_ intended?

Reply:
Yes! https://github.com/DataDog/integrations-core/pull/21266